learn/cli-commands.md

```diff
@@ -6,82 +6,119 @@ permalink: /learn/cli-commands/
+
+`ballerina <THE-COMMAND> <ITS-ARGUEMENTS>`
+
+The below are all the actions you can perform with it:
+
+- [Get started](#get-started)
+- [Build, test, and run programs](#build-test-and-run-programs)
+- [Work with Ballerina Central](#work-with-ballerina-central)
+- [Manage projects](#manage-projects)
+- [Use the supporting tools](#use-the-supporting-tools)
+
+## Get started
+These commands help you check your installation and know that things are set up smoothly.
+
+<table class="cComandTable">
+<tr>
+<td class="cCommand">`version`</td>
+<td class="cDescription">Tells you the version of Ballerina you have installed.</td>
+</tr>
+<tr>
+<td class="cCommand">home</td>
+<td class="cDescription">Prints out the location of you current Ballerina distribution. This is useful if you need to help your IDE plugin to figure out where Ballerina is.
+</td>
+</tr>
+</table>
+
+## Build, test, and run programs
+
+These everyday commands are your best friends! They address the very basics of programming in Ballerina such as compiling, running, testing programs, and generating their documentation.
+
+<table class="cComandTable">
+<tr>
+<td class="cCommand">build</td>
+<td class="cDescription">Compile a Ballerina program, a single BAL file, an entire project, or a single root module into an executable JAR file.
+</td>
+</tr>
+<tr>
+<td class="cCommand">test</td>
+<td class="cDescription">Run tests of a particular module or all the modules of a Ballerina project.
+</td>
+</tr>
+<tr>
+<td class="cCommand">run</td>
+<td class="cDescription">Build and run a Ballerina program, a single BAL file, an entire project, or a previously-built program.
+</td>
+</tr>
+<tr>
+<td class="cCommand">clean</td>
+<td class="cDescription">Clean all artifacts generated by the build command for a project.
+</td>
+</tr>
+<tr>
+<td class="cCommand">doc</td>
+<td class="cDescription">Generate API documents for all public symbols of a Ballerina module or project.
+</td>
+</tr>
+<tr>
+<td class="cCommand">format</td>
+<td class="cDescription">Format Ballerina source files as per the <a href="/learn/style-guide">Ballerina Style Guide</a>.</td>
+</tr>
+</table>
+
+## Work with Ballerina Central
+
+Ballerina Central is how you share Ballerina modules with others in a safe, secure, and dependable way.
+
+<table class="cComandTable">
+<tr>
+<td class="cCommand">search</td>
+<td class="cDescription">Search Ballerina Central for modules.
+</td>
+</tr>
+<tr>
+<td class="cCommand">pull</td>
+<td class="cDescription">Pull a module from Ballerina Central.
+</td>
+</tr>
+<tr>
+<td class="cCommand">push</td>
+<td class="cDescription">Upload a module to Ballerina Central.
+</td>
+</tr>
+</table>
+
+## Manage projects
+
+Ballerina projects are the way to organize real world Ballerina development tasks. Learn more about it in <a href="/learn/how-to-structure-ballerina-code">How to Structure Ballerina Code</a>.
```

Comment:
```suggestion
Ballerina projects are the way to organize real-world Ballerina development tasks. Learn more about it in <a href="/learn/how-to-structure-ballerina-code">How to Structure Ballerina Code</a>.
```

Reply:
The hyphen is not needed when saying "real world".

<img width="663" alt="Screenshot 2019-12-02 at 20 41 50" src="https://user-images.githubusercontent.com/11707273/69970464-4e124100-1544-11ea-9a30-a4b52bcd509a.png">